src/Client.cpp

```diff
@@ -41,6 +41,7 @@ err_code_t Client::get(const char* const* keys, const size_t* keyLens, size_t nK
   dispatchRetrieval(GET_OP, keys, keyLens, nKeys);
   err_code_t rv = waitPoll();
   collectRetrievalResult(results, nResults);
+  log_info_if(rv != RET_OK, "[I: %p], err code %s", this, errCodeToString(rv));
```

Comment:
AFAICS, it's no need to add log here, because the user can get `rv` and `client` outside.

Reply:
@tclh123 also because pointer address。